operator/forklift-operator-catalog.yaml

```diff
@@ -4,7 +4,10 @@ metadata:
   name: konveyor-forklift
   namespace: openshift-marketplace
 spec:
-  displayName: Forklift Operator
+  displayName: Liran Rotenberg
   publisher: Konveyor
   sourceType: grpc
-  image: quay.io/konveyor/forklift-operator-index:latest
+  image: quay.io/lrotenbe/forklift-operator-index:latest
+  updateStrategy:
+    registryPoll:
+      interval: 10m0s
```

Comment:
I realize the changes above are temporary but also this one?

Reply:
yup